app/Http/Controllers/ReloadlyController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Services\ReloadlyService;
+use Illuminate\Support\Facades\Log;
+
+class ReloadlyController extends Controller
+{
+    public function getBalance() {
+		Log::info("[ReloadlyService][checkAccountBalance]\t Checking Account Balance");
+	$balance_response = ReloadlyService::getBalance();
+	if (!empty($balance_response)) {
+		return [
+			"code" => 200,// Assign a code here for success you will use throughout
+			"balance" => $balance_response["balance"],
+			"currency" => $balance_response["currencyCode"],
+			"message" => "Balance retrieved successfully"
+		];
+	}
+	return [
+		"code" => 400,// Assign a code you will use for error getting balance
+		"message" => "Could not retrieve the account balance",
+	];
+
+	}
+    
+    public function getTransactionStatus($id, Request $request) { // Added the $id variable defined in the routes as a parameter to use later
+		try {
+			$result = ReloadlyService::getTransactionStatus();    
+		} catch (\Exception $exception) {
```

Comment:
Most likely, you would pass the `$id` variable into the `getTransactionStatus()` function